test/parallel/test-buffer-readuintbe.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUIntBE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUIntBE(2);
+
+assert.strictEqual(result, 84);
+
+assert.throws(
+   () => {
+     buf.readUIntBE(5);
+   }, RangeError
```

Comment:
@julianduque Can you be a bit more specific? To me, this seems OK if it's indented two spaces, which it seems to be here.

Are you saying the arrow function should be all on one line?

Or are you saying that the arrow function should start one line above, right after the parenthesis?

Or something else?